rm-datasource/src/main/java/io/seata/rm/datasource/undo/UndoExecutorFactory.java

```diff
@@ -40,31 +35,24 @@
      * @return the undo executor
      */
     public static AbstractUndoExecutor getUndoExecutor(String dbType, SQLUndoLog sqlUndoLog) {
-        if (!dbType.equalsIgnoreCase(JdbcConstants.MYSQL)&&!dbType.equalsIgnoreCase(JdbcConstants.ORACLE)) {
-            throw new NotSupportYetException(dbType);
-        }
-          if(dbType.equalsIgnoreCase(JdbcConstants.ORACLE)) {
-            switch (sqlUndoLog.getSqlType()) {
-                case INSERT:
-                    return new OracleUndoInsertExecutor(sqlUndoLog);
-                case UPDATE:
-                    return new OracleUndoUpdateExecutor(sqlUndoLog);
-                case DELETE:
-                    return new OracleUndoDeleteExecutor(sqlUndoLog);
-                default:
-                    throw new ShouldNeverHappenException();
-            }
-        } else {
-              switch (sqlUndoLog.getSqlType()) {
-                  case INSERT:
-                      return new MySQLUndoInsertExecutor(sqlUndoLog);
-                  case UPDATE:
-                      return new MySQLUndoUpdateExecutor(sqlUndoLog);
-                  case DELETE:
-                      return new MySQLUndoDeleteExecutor(sqlUndoLog);
-                  default:
-                      throw new ShouldNeverHappenException();
-              }
-          }
+      if (!dbType.equals(JdbcConstants.MYSQL) && !dbType.equals(JdbcConstants.POSTGRESQL)) {
```

Comment:
why POSTGRESQL?

Reply:
Sorry, my another local code branch has added support for PG database. I copied the code there directly, so the code here has PG judgment. At present, I should modify it to Oracle